Kitodo/src/main/java/de/sub/goobi/helper/Helper.java

```diff
@@ -507,65 +500,58 @@ public static AktuelleSchritteForm getCurrentTaskForm() {
-        @Override
-        public boolean accept(File dir, String name) {
-            boolean fileOk = false;
-            String prefix = ConfigCore.getParameter("ImagePrefix", "\\d{8}");
-
-            List<String> regexList = new ArrayList<>();
-            regexList.add(prefix + "\\.[Tt][Ii][Ff][Ff]?");
-            regexList.add(prefix + "\\.[jJ][pP][eE]?[gG]");
-            regexList.add(prefix + "\\.[jJ][pP][2]");
-            regexList.add(prefix + "\\.[pP][nN][gG]");
-            regexList.add(prefix + "\\.[gG][iI][fF]");
-
-            for (String regex : regexList) {
-                if (name.matches(regex)) {
-                    fileOk = true;
-                }
-            }
+    public static final FilenameFilter imageNameFilter = (dir, name) -> {
+        boolean fileOk = false;
+        String prefix = ConfigCore.getParameter("ImagePrefix", "\\d{8}");
 
-            return fileOk;
+        List<String> regexList = new ArrayList<>();
+        regexList.add(prefix + "\\.[Tt][Ii][Ff][Ff]?");
+        regexList.add(prefix + "\\.[jJ][pP][eE]?[gG]");
+        regexList.add(prefix + "\\.[jJ][pP][2]");
+        regexList.add(prefix + "\\.[pP][nN][gG]");
+        regexList.add(prefix + "\\.[gG][iI][fF]");
+
+        for (String regex : regexList) {
+            if (name.matches(regex)) {
+                fileOk = true;
+            }
         }
+
+        return fileOk;
     };
 
-    public static final FilenameFilter dataFilter = new FilenameFilter() {
-
-        @Override
-        public boolean accept(File dir, String name) {
-            boolean fileOk = false;
-            String prefix = ConfigCore.getParameter("ImagePrefix", "\\d{8}");
-
-            List<String> regexList = new ArrayList<>();
-            regexList.add(prefix + "\\.[Tt][Ii][Ff][Ff]?");
-            regexList.add(prefix + "\\.[jJ][pP][eE]?[gG]");
-            regexList.add(prefix + "\\.[jJ][pP][2]");
-            regexList.add(prefix + "\\.[pP][nN][gG]");
-            regexList.add(prefix + "\\.[gG][iI][fF]");
-            regexList.add(prefix + "\\.[pP][dD][fF]");
-            regexList.add(prefix + "\\.[aA][vV][iI]");
-            regexList.add(prefix + "\\.[mM][pP][gG]");
-            regexList.add(prefix + "\\.[mM][pP]4");
-            regexList.add(prefix + "\\.[mM][pP]3");
-            regexList.add(prefix + "\\.[wW][aA][vV]");
-            regexList.add(prefix + "\\.[wW][mM][vV]");
-            regexList.add(prefix + "\\.[fF][lL][vV]");
-            regexList.add(prefix + "\\.[oO][gG][gG]");
-            regexList.add(prefix + "\\.docx");
-            regexList.add(prefix + "\\.xls");
-            regexList.add(prefix + "\\.xlsx");
-            regexList.add(prefix + "\\.pptx");
-            regexList.add(prefix + "\\.ppt");
-
-            for (String regex : regexList) {
-                if (name.matches(regex)) {
-                    fileOk = true;
-                }
+    public static final FilenameFilter dataFilter = (dir, name) -> {
```

Comment:
Missing JavaDoc for public method. Or move this code into its own class to get ride out of this inner class usage.

Reply:
JavaDocs can't be made for lambdas - param attributes are not allowed there - and it is constant.